docs/docs/metrics.md

```diff
@@ -4,112 +4,89 @@ title: Metrics
 
 ## Important metrics
 
-`service.mesosphere.marathon.uptime` (gauge) - The uptime of the reporting Marathon process
-in milliseconds. This is helpful to diagnose stability problems which cause
-Marathon to restart.
+`service.mesosphere.marathon.uptime` (gauge) - The uptime of the reporting Marathon process in milliseconds. This is helpful to diagnose stability problems that cause Marathon to restart.
 
-### App/group/task counts
+### App, group, and task counts
 
-`service.mesosphere.marathon.app.count` (gauge) - The number of defined apps. In general,
-this is one number which influences the performance of Marathon. If you have
+`service.mesosphere.marathon.app.count` (gauge) - The number of defined apps. This number influences the performance of Marathon: if you have
 a high number of apps, your performance will be lower than for a low number of
 apps.
 
-`service.mesosphere.marathon.group.count` (gauge) - The number of defined groups. In general,
-this is one number which influences the performance of Marathon. If you have
-a high number of groups, your performance will be lower than for a low number of
-groups. Note that each level between the slashes in your app IDs corresponds to
-a group. The app `/shop/frontend` is in the `frontend` group which is in
-the `shop` group which is in the root group.
+`service.mesosphere.marathon.group.count` (gauge) - The number of defined groups. This number influences the performance of Marathon: if you have a high number of groups, your performance will be lower than for a low number of groups. Note that each term between the slashes in your an ID corresponds to a group. The app `/shop/frontend` is in the `frontend` group, which is in the `shop` group, which is in the root group.
 
 <span class="label label-default">v0.15</span>
 `service.mesosphere.marathon.task.running.count` (gauge) - The number of tasks that are
 currently running.
 
-<span class="label label-default">v0.15</span>
+<span class="label label-default">v0.15</span> 
 `service.mesosphere.marathon.task.staged.count` (gauge) - The number of tasks that are
-currently staged. After tasks are launched, they are first in this state.
-A consistently high number of staged tasks indicates a lot of churn in Marathon
-and Mesos. Either you have many app updates/manual restarts or some of your apps
-have stability problems and are consistently automatically restarted.
+currently staged. Tasks enter staging state after they are launched. A consistently high number of staged tasks indicates a lot of churn in Marathon and Mesos. Either you have many app updates/manual restarts or some of your apps have stability problems and are automatically restarted frequently.
```

Comment:
I am not a native speaker, but it describes the fact, that a lot of tasks terminate and get restarted. Any better word for this?